catalog/task/create-pyxis-image/0.1/create-pyxis-image.yaml

```diff
@@ -10,7 +10,7 @@ metadata:
     tekton.dev/tags: release
 spec:
   description: >-
-    Tekton task that executes a python script to push a container image's metadata to Pyxis
+    Tekton task that pushes metadata for all container images contained in a snapshot to Pyxis
```

Comment:
```suggestion
    Tekton task that pushes to Pyxis metadata for all container images contained in a snapshot
```
I think it reads better.

Reply:
I disagree, I think it's bad English - the original was the proper order IMHO. But Johnny is in a better position to judge, being a native speaker :)

Reply:
I think they all aren't great tbh haha

Reply:
To pyxis at end reads better imho

Reply:
Yeah, actually I wanted to write `Tekton task that pushes metadata to Pyxis for all container images contained in a snapshot`.

Reply:
> Yeah, actually I wanted to write `Tekton task that pushes metadata to Pyxis for all container images contained in a snapshot`.

I am good with that. Objections?

Reply:
I think it's good now :) Sorry for the late response.